### ISSUE TITLE
Figure out a way to trigger build in judgels when a commit is pushed to its submodules

### ISSUE THREAD

**Issue by fushar** [ia-toki/judgels#67]
This is for integration tests over the whole Judgels applications.

**Comment by petrabarus** [comment]
I think you have to add webhook in each submodule repo for this, right?

**Comment by fushar** [comment]
Yes, but I think this requires us to setup a web server that handles the hook, right? Is there any GitHub-only solution?

**Comment by petrabarus** [comment]
where do you do the build right now?

**Comment by fushar** [comment]
To be honest, currently we don't use the feature -- only grading types have unit tests at the moment and they are run locally
